WidgetKit/ImageButtonWithLabel.swift

```diff
@@ -75,11 +75,13 @@ struct ImageButtonWithLabel: View {
                                     .font(.headline)
                                     .minimumScaleFactor(0.75)
                                     .layoutPriority(1000)
+                                    .foregroundColor(link.textColor)
                             } else {
                                 Text(link.label)
                                     .font(.footnote)
                                     .minimumScaleFactor(0.75)
                                     .layoutPriority(1000)
+                                    .foregroundColor(link.textColor)
```

Comment:
Shouldn't we add `// Ecosia: add color` like below?

Reply:
@lucaschifino thanks for the reminder! Has slipped thru the tasks 🙏